womtool/src/main/scala/cromwell/pipeline/WomTool.scala

```diff
@@ -0,0 +1,85 @@
+package cromwell.pipeline
+
+import cats.data.NonEmptyList
+import com.typesafe.config.ConfigFactory
+import common.Checked
+import cromwell.core.path.{ DefaultPath, DefaultPathBuilder, Path }
+import cromwell.languages.LanguageFactory
+import cromwell.languages.util.ImportResolver.{ DirectoryResolver, HttpResolver, ImportResolver }
+import cromwell.pipeline.WomTool7.wdlFileContent
+import languages.cwl.CwlV1_0LanguageFactory
+import languages.wdl.biscayne.WdlBiscayneLanguageFactory
+import languages.wdl.draft2.WdlDraft2LanguageFactory
+import languages.wdl.draft3.WdlDraft3LanguageFactory
+import wom.executable.WomBundle
+import womtool.WomtoolMain.{ BadUsageTermination, SuccessfulTermination, Termination, UnsuccessfulTermination }
+import womtool.inputs.Inputs
+
+import scala.util.Try
+
+class WomTool extends WomToolAPI {
+
+  def validate(
+    content: String,
+    importResolver: Any
+  ): Either[NonEmptyList[String], (WomBundle, LanguageFactory)] = {
+
+    lazy val importResolvers: List[ImportResolver] = //TODO - deal with import resolvers
+      DirectoryResolver.localFilesystemResolvers(None) :+ HttpResolver(relativeTo = None)
+    val languageFactory: LanguageFactory =
+      List(
+        new WdlDraft3LanguageFactory(ConfigFactory.empty()),
+        new WdlBiscayneLanguageFactory(ConfigFactory.empty()),
+        new CwlV1_0LanguageFactory(ConfigFactory.empty())
+      ).find(_.looksParsable(content)).getOrElse(new WdlDraft2LanguageFactory(ConfigFactory.empty()))
```

Comment:
Why do we call ConfigFactory.empty() four times? Isn't that the right way to create val for this?

Reply:
This code uses Cromwell logic as it is